packages/server/server/src/server/devServerOld.ts

```diff
@@ -0,0 +1,462 @@
+import { createServer, IncomingMessage, Server, ServerResponse } from 'http';
```

Comment:
remove this file ? and also remove the dependencies like "http-compression"

Reply:
The old modern server and deps should be replaced by ModernRsbuildServer after rsbuild is fully integrated into modern.js. Now, we still need it.